src/mime.cr

```diff
@@ -1,31 +1,30 @@
 require "json"
 
 module Mime
+  MIME_JSON = {{ `cat #{__DIR__}/types.json`.stringify }}
```

Comment:
Is there a way we can do this without relying on `cat` or inline shell execution?

Reply:
I was following https://github.com/crystal-lang/crystal/issues/1649 in using `cat`. I could change it to https://github.com/schovi/baked_file_system if you'd like :smile:

Reply:
I'd feel more comfortable with baked_file_system so that we aren't relying on `cat` being available. Do you think baked_file_system is too heavy a dependency to accomplish this task? Alternatively we could save the types.json file as a .cr file or something like that, since it's generated anyway.

Reply:
> Do you think baked_file_system is too heavy a dependency to accomplish this task?

I gave it a quick read; it looks relatively lightweight (and doesn't have any dependencies). I'll modify this PR tonight or tomorrow to add it.